package.json

```diff
@@ -17,15 +18,24 @@
     "url": "https://github.com/emote-chat/server/issues"
   },
   "dependencies": {
+    "bcryptjs": "^2.4.3",
     "bluebird": "^3.5.4",
     "body-parser": "^1.18.3",
     "dotenv": "^7.0.0",
     "express": "^4.16.4",
+    "express-jwt": "^5.3.1",
+    "jsonwebtoken": "^8.5.1",
     "path": "^0.12.7",
-    "pg": "^7.9.0",
     "pg-promise": "^8.6.5"
   },
   "devDependencies": {
-    "jshint": "^2.10.2"
```

Comment:
Didn't mention this, but I figured we don't care about `jshint` anymore. Maybe someone else can hook up `eslint` if  they care enough, but as far as tests go, I think we'll want to run unit tests for the API and verify they pass with Circle CI. I'll have to see if I need to do more configuration to run the tests with Circle CI.